documentation/staging/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -719,7 +719,7 @@ It is important to avoid applying unsupported model updates to a running domain.
 
 The following summarizes the types of runtime update configuration that are _not_ supported in Model in Image unless a workaround or alternative is documented:
 
-  - Decreasing dynamic cluster size:
+  - Altering cluster size:
```

Comment:
This content is specific to making the cluster size smaller. The same concerns do not apply to making the cluster size larger.

Reply:
@rjeberhard Interesting. This is a correction that was given to me by @tbarnes-us: "Decreasing dynamic cluster size" should be "Altering cluster size"

Reply:
Okay 🤷

Reply:
The content is not specific to increase/decrease. You can increase the size of a dynamic cluster now IIRC, but not a configured cluster - at least that's what we document under the bullet.